packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-restricted-imports */
+import { useHelpCenterMessenger } from '@automattic/help-center/src/components/help-center-messenger';
```

Comment:
Should probably export it from the package and access it like `@automattic/help-center`